modules/swagger-codegen/src/main/java/io/swagger/codegen/InlineModelResolver.java

```diff
@@ -363,18 +375,45 @@ public Model modelFromProperty(MapProperty object, @SuppressWarnings("unused") S
         String example = null;
 
         Object obj = object.getExample();
-        if(obj != null) {
+        if (obj != null) {
             example = obj.toString();
         }
 
         ArrayModel model = new ArrayModel();
         model.setDescription(description);
         model.setExample(example);
         model.setItems(object.getAdditionalProperties());
+        this.copyVenderExtensions(object, model);
 
         return model;
     }
 
+    /**
+     * Copy vender extensions from Property to Model
+     * 
+     * @param source
+     * @param target
+     */
+    private void copyVenderExtensions(Property source, AbstractModel target) {
+        Map<String, Object> venderExtensions = source.getVendorExtensions();
+        for (String extName : venderExtensions.keySet()) {
+            target.setVendorExtension(extName, venderExtensions.get(extName));
+        }
+    }
```

Comment:
@zhengcan thanks for the PR. Can you tell me more on why we need to copy the vendor extension from a property to the property's model itself?